core/imt/inc/ROOT/TExecutor.hxx

```diff
@@ -161,8 +160,7 @@ private:
       std::vector<retType> res;;
       switch(fExecPolicy){
          case ROOT::Internal::ExecutionPolicy::kSerial:
-            //arbitrary value for the number of chunks so the returned vector is not big
-            res = fSeqPool->Map(func, nTimes, redfunc, 3);
+            res = fSeqPool->Map(func, nTimes, redfunc, 1);
```

Comment:
What are those magic numbers? Could one explicitly enumerate the states in ExecutionPolicy, and `static_cast` those states into the numbers?
Or did I miss something, and it's something completely different?

Reply:
Probably need to comment, or just not pass those magic numbers (the number of chunks in a sequential executor, which don't matter).